### ISSUE TITLE
Cant upgrade to 6.0.0 because of missing dependency RN 0.60.0. I´m using RN 0.63.4

### ISSUE THREAD

**Issue by Finneah** [mockingbot/react-native-zip-archive#234]
**Describe the bug**
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: myproject@1.0.0
npm ERR! Found: react-native@0.63.4
npm ERR! node_modules/react-native
npm ERR!   react-native@"^0.63.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react-native@"^0.60.0" from react-native-zip-archive@6.0.0
npm ERR! node_modules/react-native-zip-archive
npm ERR!   react-native-zip-archive@"6.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See ... for a full report.

npm ERR! A complete log of this run can be found in:
...
```
**To Reproduce**
my version of  react-native-zip-archive: `^5.0.2`
1. Cmd`npm i react-native-zip-archive@6.0.0`

**Expected behavior**

**Env (please complete the following information):**
- Npm 7.5.2
- RN 0.63.4 
- Package Version: 5.0.2

**Additional context**

**Comment by yfunk** [comment]
If you run `npm i react-native-zip-archive@6.0.0 --legacy-peer-deps` like the error suggest it will work.

The reason for this error is, that `react-native-zip-archive` currently specifies `"react-native": "^0.60.0"` in its `peerDependencies`. You would think this means version 0.60.0 or newer needs to be installed, but because the major version of `react-native` is still at `0` only patch updates are allowed, which means this package actually specifies it needs version `0.60.x` of React Native installed. At the same time this error is not happening for the `react` peer dependency, because it is at major version `16`. [More details on caret ranges](https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004). 

The installation will only fail because of this error at npm version 7 or newer (unless you specify `--legacy-peer-deps`).

The fix for this should be as easy as updating `peerDependencies` like this:
```
"peerDependencies": {
    "react": ">=0.60.0",
    "react": "^16.8.6",
  }
```

**Comment by plrthink** [comment]
@plrthink @Finneah just published 6.0.1 which should fix this issue for npm@7
